test/rekt/features/pingsource/features.go

```diff
@@ -18,7 +18,7 @@ package pingsource
 
 import (
 	"context"
-	
+
```

Comment:
can you rebase your PR?

See: https://github.com/knative/eventing/commit/d547e2d30f39a7bf5c008db394e6bbb791b11ae4